index.html

```diff
@@ -251,6 +252,37 @@ <h4>Skills</h4>
             </ul>
         </div>
        </section>
+    </section>  
+
+    <section id="contact-form">
+        <p>I'm always interested in hearing about new projects, so if you you'd like to chat please get in touch</p>
+        <div class="inputs">
+            <form class="contact" action="https://formspree.io/f/mgebgadd" method="post">
+                <input class="form1" type="text" id="name" type="text" placeholder="Full name" maxlength="30"   required ><br>
+                <input class="form2" name="Email" id="email" type="email" placeholder="Email address"   required><br>
+                <textarea class="text_form" name="textarea" id="textarea" placeholder="Hey I'd like to connect to..." maxlength="50"  required></textarea><br>
```

Comment:
Well done so far 💪. However, I think that a max-length value of 50 is too small for a ```text area `` tag don't you think 😉. I think that making the max-length about 500 would be a better fit for a ```text area `` tag so that your user has enough room to send you a detailed enough message.